marimo/_output/formatters/arviz_formatters.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+from typing import Any, Union
+from marimo._messaging.mimetypes import KnownMimeType
+from marimo._output.formatters.formatter_factory import FormatterFactory
+import matplotlib.pyplot as plt
+import numpy as np
+
+class ArviZFormatter(FormatterFactory):
+    @staticmethod
+    def package_name() -> str:
+        return "arviz"
+
+    def register(self) -> None:
+        import arviz as az
```

Comment:
you can re-import matplitlib and np here

Reply:
Done.